test/validators/movies/list.test.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const Joi = require('joi');
+
+const ListMoviesValidator = require('../../../lib/validators/movies/list');
+
+describe('list movies validator', () => {
+
+  describe('title', () => {
+
+    it('is a string', () => {
+      const payload = { title: 'The Dark Knight' };
+      const result = Joi.validate(payload, ListMoviesValidator);
+
+      expect(result.error).to.be.null;
```

Comment:
I would advise against checking `null`. The reason being is because really what we are trying to test here is the absence of an error, not necessarily that the error is actually `null`. The assertion that would be a better fit is [`exist`](http://chaijs.com/api/bdd/#method_exist).

Across different libraries (and even within the same library), `undefined` and `null` are used interchangeably to represent a non-existent value. As such, it isn't always safe to specifically check for one of them when, from the library author's perspective, either value could work.